index.md

```diff
@@ -12,7 +12,7 @@ Welcome to the SlimeVR documentation. This site covers how to [build your own Sl
 
 ## So you want a set of SlimeVR trackers, what are the options?
 
-![](https://i.imgur.com/dLJBuM9.jpg)
+![](https://i.imgur.com/dLJBuM9.jpg)<br>
```

Comment:
Why use HTML tag, when you can just add a line break?
```suggestion
![](https://i.imgur.com/dLJBuM9.jpg)

```

Reply:
Line break wasn't stopping the wrap around on my local server